src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -158,7 +158,7 @@
             class="tc_submenuitem tc_submenuitem_monitoring"
             *ngIf="permissions.prometheus.read">
           <a routerLink="/monitoring">
-            <ng-container i18n>Monitoring</ng-container>
+            <ng-container i18n>Alerts</ng-container>
```

Comment:
`<ng-container>` is meant to be a placeholder for `*ng` directives, so if there is no directive let's use an appropriate element like `<div>` or `<span>`.
In this file we should replace all the imporper uses of `<ng-container>`.
```suggestion
            <span i18n>Alerts</ng-container>
```